### ISSUE TITLE
Demos broken?

### ISSUE THREAD

**Issue by corysimmons** [azazdeaz/react-gsap-enhancer#37]
They don't seem to be doing anything.

**Comment by azazdeaz** [comment]
They are working for me in the latest Chrome and Firefox. In which browser do you have this issue?

**Comment by corysimmons** [comment]
Latest Chrome, Mac OS Sierra.

UOrigin AdBlock, HTTPS Everywhere, Ghostery.

**Comment by corysimmons** [comment]
Yeah looks like http:// in your cdnjs requests is causing a cross-origin problem.
